apply_new_tech_terraform.Jenkinsfile

```diff
@@ -129,14 +129,14 @@ pipeline {
     parameters {
         string(name: 'CONFIG_BRANCH', description: 'Target Branch for hmpps-env-configs', defaultValue: 'master')
         string(name: 'NEWTECH_BRANCH', description: 'Target Branch for hmpps-delius-new-tech-terraform', defaultValue: 'master')
-        booleanParam(name: 'deploy_NTCaseNotes', defaultValue: true, description: 'Deploy New Tech Case Notes?')
-        booleanParam(name: 'deploy_NTPDFGenerator', defaultValue: true, description: 'New Tech PDF Generator?')
-        booleanParam(name: 'deploy_NTOffenderAPI', defaultValue: true, description: 'New Tech Offender API?')
-        booleanParam(name: 'deploy_NTElasticSearch', defaultValue: true, description: 'New Tech ElasticSearch?')
+        booleanParam(name: 'deploy_NTCaseNotes', defaultValue: false, description: 'Deploy New Tech Case Notes?')
+        booleanParam(name: 'deploy_NTPDFGenerator', defaultValue: false, description: 'New Tech PDF Generator?')
+        booleanParam(name: 'deploy_NTOffenderAPI', defaultValue: false, description: 'New Tech Offender API?')
+        booleanParam(name: 'deploy_NTElasticSearch', defaultValue: false, description: 'New Tech ElasticSearch?')
         booleanParam(name: 'deploy_NTDOffenderSearch', defaultValue: false, description: 'New Tech Offender Search Service?')
-        booleanParam(name: 'deploy_NTOffenderPollPush', defaultValue: true, description: 'New Tech Offender Poll Push?')
-        booleanParam(name: 'deploy_NTWebFrontend', defaultValue: true, description: 'New Tech Web Frontend?')
-        booleanParam(name: 'deploy_NTDashboards', defaultValue: true, description: 'New Tech Dashboards?')
+        booleanParam(name: 'deploy_NTOffenderPollPush', defaultValue: false, description: 'New Tech Offender Poll Push?')
+        booleanParam(name: 'deploy_NTWebFrontend', defaultValue: false, description: 'New Tech Web Frontend?')
+        booleanParam(name: 'deploy_NTDashboards', defaultValue: false, description: 'New Tech Dashboards?')
     }
```

Comment:
setting jenkins defaults to false as deployments generally require unticking of all these